src/ljs_input.cpp

```diff
@@ -107,8 +107,8 @@ int main(int argc, char **argv)
     }
     else{
       msg.trig_l_val = GamepadTriggerLength(controller, TRIGGER_LEFT) - 0.435;
-      msg.trig_l_val = (msg.trig_l_val>0.0) ? msg.trig_l_val/(1-0.435): msg.trig_l_val/(0.435);
-      if (abs(msg.trig_l_val) < 0.01) {
+      msg.trig_l_val = (msg.trig_l_val>0.0) ? msg.trig_l_val/(1-0.435): msg.trig_l_val/(0.435); //Re-scale offset value
```

Comment:
This line of code is repeated a couple of times in both files so could maybe be mapped to a super simple function to clean it up a bit but obviously not super essential

Reply:
Yeah this is a good point, although it's only used twice in each script, it's one line of code. If there was already a file that contained common functions between the two then it would be chill. Raises a good point though, there's alot of unnecessary mapped buttons in these files, we could strip them and put them into one file in the future and have all the common functions on the one file, thus we wouldn't need to ever do multiple changes...so I think that's defs a worth being a task for anyone that wants to give it a go.